examples/python/pincell_multigroup/build-xml.py

```diff
@@ -21,44 +21,47 @@
 # Instantiate the 7-group (C5G7) cross section data
 uo2_xsdata = openmc.XSdata('UO2', groups)
 uo2_xsdata.order = 0
-uo2_xsdata.total = [0.1779492, 0.3298048, 0.4803882, 0.5543674,
-                    0.3118013, 0.3951678, 0.5644058]
-uo2_xsdata.absorption = [8.0248E-03, 3.7174E-03, 2.6769E-02, 9.6236E-02,
-                         3.0020E-02, 1.1126E-01, 2.8278E-01]
-uo2_xsdata.scatter = [[[0.1275370, 0.0423780, 0.0000094, 0.0000000, 0.0000000, 0.0000000, 0.0000000],
-                       [0.0000000, 0.3244560, 0.0016314, 0.0000000, 0.0000000, 0.0000000, 0.0000000],
-                       [0.0000000, 0.0000000, 0.4509400, 0.0026792, 0.0000000, 0.0000000, 0.0000000],
-                       [0.0000000, 0.0000000, 0.0000000, 0.4525650, 0.0055664, 0.0000000, 0.0000000],
-                       [0.0000000, 0.0000000, 0.0000000, 0.0001253, 0.2714010, 0.0102550, 0.0000000],
-                       [0.0000000, 0.0000000, 0.0000000, 0.0000000, 0.0012968, 0.2658020, 0.0168090],
-                       [0.0000000, 0.0000000, 0.0000000, 0.0000000, 0.0000000, 0.0085458, 0.2730800]]]
-uo2_xsdata.fission = [7.21206E-03, 8.19301E-04, 6.45320E-03,
-                      1.85648E-02, 1.78084E-02, 8.30348E-02,
-                      2.16004E-01]
-uo2_xsdata.nu_fission = [2.005998E-02, 2.027303E-03, 1.570599E-02,
-                         4.518301E-02, 4.334208E-02, 2.020901E-01,
-                         5.257105E-01]
-uo2_xsdata.chi = [5.8791E-01, 4.1176E-01, 3.3906E-04, 1.1761E-07,
-                  0.0000E+00, 0.0000E+00, 0.0000E+00]
+uo2_xsdata.set_total(
```

Comment:
I liked the old property setters, but I suppose I'll find out why you now use `set_*` methods below